iina/MainWindowController.swift

```diff
@@ -2744,13 +2648,9 @@ class MainWindowController: NSWindowController, NSWindowDelegate {
 
   // MARK: - Utility
 
-  func handleIINACommand(_ cmd: IINACommand) {
-    let appDeletate = (NSApp.delegate! as! AppDelegate)
+  internal override func handleIINACommand(_ cmd: IINACommand) {
```

Comment:
It's not something new, but do we need to use `self` in this function?

Reply:
No.